src/preact/index.js

```diff
@@ -65,8 +65,9 @@ export function createRef() {
 }
 
 /**
- * @param {!Object} value
- * @return {!PreactDef.Context}
+ * @param {T=} value
```

Comment:
Is there a reason this was required before?

Reply:
Oops, missed this one too.